libs/two/valup/src/lib/environments/environment.prod.ts

```diff
@@ -0,0 +1,7 @@
+export const environment = {
+  production: true
+};
+
+export const api_url = `${window.location.protocol}//${window.location.hostname}/geoservices.tamu.edu/rest/`;
+
+export const covid_api_url = `https://nodes.geoservices.tamu.edu/api/covid/`;
```

Comment:
Application environment configs should not exist in libraries.

`covid_api_url` not needed.